requirements.txt

```diff
@@ -0,0 +1,5 @@
+requests
+jira
+mock
```

Comment:
I do not think you need this one.

Reply:
had to add it as some tests used it,
if it won't be needed at the end i will remove,
without it the `import mock` failed